app/models/site.rb

```diff
@@ -1,4 +1,19 @@
 class Site < Base
   has_one :site_status
-  belongs_to :provider
+  belongs_to :provider, param: :provider_code
+
+  REGIONS = [
+    ['', :no_region],
```

Comment:
Should this be explicitly "No region" @benilovj?

Reply:
👍 to this suggestion